lib/ui/views/installer/installer_view.dart

```diff
@@ -22,24 +22,26 @@ class InstallerView extends StatelessWidget {
           child: Scaffold(
             floatingActionButton: Visibility(
               visible: !model.isPatching && !model.hasErrors,
-              child: FloatingActionButton.extended(
-                label: I18nText(
-                  model.isInstalled
-                      ? 'installerView.openButton'
-                      : 'installerView.installButton',
-                ),
-                icon: model.isInstalled
-                    ? const Icon(Icons.open_in_new)
-                    : const Icon(Icons.file_download_outlined),
-                onPressed: model.isInstalled
-                    ? () => {
-                          model.openApp(),
-                          model.cleanPatcher(),
-                          Navigator.of(context).pop(),
-                        }
-                    : () => model.installTypeDialog(context),
-                elevation: 0,
-              ),
+              child: model.isInstalled
+                  ? FloatingActionButton.extended(
+                      label: I18nText('installerView.openButton'),
+                      icon: const Icon(Icons.open_in_new),
+                      onPressed: () {
+                        model
+                          ..openApp()
+                          ..cleanPatcher();
+                        Navigator.of(context).pop();
+                      },
+                      elevation: 0,
+                    )
+                  : FloatingActionButton.extended(
+                      label: I18nText('installerView.installButton'),
+                      icon: const Icon(Icons.file_download_outlined),
+                      onPressed: model.isInstalling
+                          ? null
```

Comment:
It's been a while since I didn't write anything in Flutter (3 months or sth). AFAIK, passing `null` to `onPressed` should also grey-out its styling, but this does not, for some reason (new Metarial 3 thing I suppose).

It behaves, tho.

Reply:
FABs can't be disabled

Reply:
@Ushie I can apply a custom disabled styling or hide FAB altogether. Would you like me to? :)

Reply:
Hi, sorry for the late reply

By material design guidelines, FABs can be hidden but not disabled, so it'd certainly be better to go with hiding it

Reply:
Thanks for the feedback. Will apply the patch as soon as I'm available. :)

Reply:
Applied the patch.